Dockerfile

```diff
@@ -59,12 +70,14 @@ WORKDIR /app
 # Copy built executable and any staged resources from builder
 COPY --from=build --chown=vapor:vapor /staging /app
 
+# Provide configuration needed by the built-in crash reporter and some sensible default behaviors.
+ENV SWIFT_BACKTRACE=enable=yes,sanitize=yes,threads=all,images=all,interactive=no
+
 # Ensure all further commands run as the vapor user
 USER vapor:vapor
 
 # Let Docker bind to port 8080
 EXPOSE 8080
 
-# Start the Vapor service when the image is run, default to listening on 8080 in production environment
-ENTRYPOINT [ "./Penny" ]
-CMD [ "serve", "--env", "production", "--hostname", "0.0.0.0", "--port", "8080" ]
```

Comment:
`CMD` is useful for Vapor apps but doesn't do anything for Penny as it's not a Vapor app.